src/main/scala/viper/silver/parser/FastParser.scala

```diff
@@ -34,12 +34,21 @@ object FastParser extends PosParser[Char, String] {
    */
   val standard_import_directory = "import"
 
-  var _lines: Array[Int] = null
+  var _line_offset: Array[Int] = null
 
   def parse(s: String, f: Path, plugins: Option[SilverPluginManager] = None) = {
     _file = f.toAbsolutePath
+
+    // Add an empty line at the end to make `computeFrom(s.length)` return `(lines.length, 1)`, as the old
+    // implementation of `computeFrom` used to do.
     val lines = s.linesWithSeparators
-    _lines = lines.map(_.length).toArray
+    _line_offset = (lines.map(_.length) ++ Seq(0)).toArray
```

Comment:
As described in the comment above, the ` ++ Seq(0)` here is used to accurately reproduce an edge case of the old behavior. However, It's not clear to me how important that edge case is. Maybe we can ignore it.